src/components/dataTables/table/index.tsx

```diff
@@ -21,17 +21,18 @@ export interface Props {
   children?: React.ReactNode
   rows?: Row[]
   rowTheme?: {[key: string]: string}
+  customStyle?: {[key: string]: string}
```

Comment:
cc @petemill and @cezaraugusto as it changes common component

Reply:
After conferring with @cezaraugusto and @petemill, removed modifications to table and tabs component